### ISSUE TITLE
Store last timestamp in JS and poll server only for new messages (not entire queue) 

### ISSUE THREAD

**Issue by danamodio** [stefanaspect/microchat#5]

**Comment by stefanaspect** [comment]
API actually already has date, as time stamped from the server side, within it, so could just use that. Indeed, this is what the app uses for determining which posts from the queue to actually display. I got lazy when I wrote the original, but it's a good idea to add.

**Comment by danamodio** [comment]
Yeah, JS just needs to query with ?since=[ts], and server iterate w/ sub-queue. No [ts], then whole queue. Could increase scroll-back and polling frequency.

**Comment by stefanaspect** [comment]
yep yep, exactly what I was thinking. Dunno about polling frequency (that might be better served by just doing actual long poll with websockets), but the idea is sound.
